nj/card.py

```diff
@@ -4,6 +4,7 @@
 from label import parse_labels
 from trelloutil import format_due_date, backlog_board, parse_new_due_date
 from worklist import sort_list
+from conf import *
```

Comment:
I'd like to steer away from wildcard imports. What each module uses from conf should be explicitly imported.